src/transforms/ppmt.jl

```diff
@@ -0,0 +1,171 @@
+using EmpiricalCDFs
+using Distributions
+using Optim
+
+"""
+    Ppmt()
+
+Applies the projection pursuit multivariate transform to the table. 
+Taken out from the documentation: Friedman, J. H. (1987). 
+Exploratory Projection Pursuit. Journal of the American Statistical 
+Association, 82(397), 249–266. https://doi.org/10.2307/2289161
+
```

Comment:
Please follow the citation format of other transforms. You also need to add appropriate indentation so that the docstring is displayed correctly in the Julia REPL.